src/import/FSHImporter.ts

```diff
@@ -996,6 +1008,37 @@ export class FSHImporter extends FSHVisitor {
     return contexts;
   }
 
+  visitCharacteristics(ctx: pc.CharacteristicsContext): LogicalCharacteristic[] {
+    const characteristics: LogicalCharacteristic[] = [];
+    ctx.CODE_ITEM().forEach(codeCtx => {
+      const characteristicCode = codeCtx.getText().slice(0, -1).trim().slice(1);
+      if (isLogicalCharacteristic(characteristicCode)) {
+        characteristics.push(characteristicCode);
+      } else {
+        logger.error(
+          'Invalid Characteristic. Supported characteristic codes are "#has-target", "#has-range", "#is-continuous", "#has-length", "#has-size", "#can-bind", "#has-units", "#do-translations", and "#can-be-target".',
```

Comment:
This is also just a question/opinion thing, but I wonder if it is worth including the codes in a defined array that could be `.join()`'ed and used in these two error messages, and maybe in the `isLogicalCharacteristic` function as well, in case the codes ever change.

Reply:
If we have to hard-code them, that's probably a good approach to localize the values to one place, but... I do think we should consider validating only in the exporter -- and we could dynamically draw the list of valid codes from the code system in that case and still log an error telling the user allowed values. I do like how helpful this message is.

Reply:
I think you're right, both that we should avoid hard-coding the list, and that it's fine to only validate on the export side. In that case, I'll update the message on the export side to give a listing of the supported codes (with a little extra woozeling to make sure that `can-be-target` appears in the message, even if it's not in the code system.)